test/methods/centroid.jl

```diff
@@ -104,7 +104,7 @@ end
     ])
     c1, area1 = GO.centroid_and_area(m1)
     c1_from_LG = LG.centroid(m1)
-    @test c1[1] ≈ GI.x(c1_from_LG)
```

Comment:
Ah is this because the points are now `GI.Point`?  We should still allow getindex behaviour on those IMO @rafaqz, just for convenience's sake.

Reply:
Yep. Would that be implemented in `GI`?